src/NationalInstruments.Analyzers/Properties/Resources.resx

```diff
@@ -414,6 +414,16 @@
   <data name="NI1816X_Title" xml:space="preserve">
     <value>Disposable should not have finalizer</value>
   </data>
+  <data name="NI1019_Description" xml:space="preserve">
+    <value>The default equality method implementation for records is to compare enumerable types by reference, not the element values;  thus it is important for records with enumerable properties to override the default implementation and compare the enumerables using SequenceEqual or UnorderedEquals (depending on whether ordering matters).</value>
+  </data>
+  <data name="NI1019_Message" xml:space="preserve">
+    <value>Record '{0}' has properties that are enumerable collections, but does not override the default Equals and GetHashCode methods</value>
```

Comment:
Since we're only checking for `Equals`, maybe we should remove `GetHashCode` from this message. Although it'd be really weird, they could have already overrode the default implement for `GetHashCode`, right?

Reply:
>they could have already overrode the default implement for GetHashCode, right?

I originally wanted to mention both `Equals` and `GetHashCode` to avoid the developer implementing `Equals` only to later find out from a different warning that they needed to override `GetHashCode` too.

But it's true that developers could override `GetHashCode` but not `Equals`—I discovered that you do *not* get a warning in that case—and find it a bit odd about the message.  However the relationship between `Equals` and `GetHashCode` is not very well known among less experienced developers, and if the developer knows to override `GetHashCode` already, then it's highly likely the developer would know to implement `Equals` too.

Perhaps I can reword it to `does not override *both* the default Equals and GetHashCode methods` (emphasis mine)?

Reply:
I reworded the message, let me know if that looks good or not.